src/screens/Grommet.js

```diff
@@ -187,11 +187,12 @@ const GrommetPage = () => (
         <Property name="themeMode">
```

Comment:
```suggestion
        <Property stable name="themeMode">
```
Can we add the stable prop to this? This is something new we are doing. The stable prop just lets people know that this is a new prop that hasn't been released yet. Once we release a new version of grommet we go through and remove all the stable flags from the docs.

Reply:
Sure thing! I will take that in mind for any future updates.